src/Components/test/testassets/Components.TestServer/RazorComponentEndpointsStartup.cs

```diff
@@ -160,6 +160,11 @@ private static void MapEnhancedNavigationEndpoints(IEndpointRouteBuilder endpoin
             return Results.Ok(value);
         });
 
+        endpoints.Map("/forms/endpoint-that-never-finishes-rendering", (HttpResponse response, CancellationToken token) =>
+        {
+            return Task.Delay(Timeout.Infinite, token);
```

Comment:
Might this have any negative effects on running the E2E tests? The server will permanently think there's an ongoing request.

It's not that I have specific knowledge this will cause any problems. Just checking you think the server will still gracefully exit at the right time.

If necessary this could be changed to just delay for 10 seconds, say, since that would easily be enough to show the behavior we're testing.

Reply:
No, it will actually get cancelled when the connection is aborted by the browser, that's why I am passing the cancellation token.